dev-python/mpmath/mpmath-1.1.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+PYTHON_COMPAT=( python{3_6,3_7,3_8,3_9} )
 
 inherit distutils-r1 eutils
 
```

Comment:
Make tests `||die`, and make sure that their custom wrapper exits with correct status. Why not use virtx?